osc-staging.py

```diff
@@ -73,6 +73,8 @@ def _full_project_name(self, project):
               help='use the old check algorithm')
 @cmdln.option('-v', '--version', action='store_true',
               help='show version of the plugin')
+@cmdln.option('--debug', action='store_true',
+              help='enable debug output')
 def do_staging(self, subcmd, opts, *args):
```

Comment:
We can use the -d option from osc itself. The debug variable can be tested from osc.conf.config